resources/metadata/helmExecute.yaml

```diff
@@ -79,14 +86,17 @@ spec:
           - STAGES
           - STEPS
       - name: targetRepositoryUser
-        description: "Username for the char repository where the compiled helm .tgz archive shall be uploaded - typically provided by the CI/CD environment."
+        description: "Username for the chart repository where the compiled helm .tgz archive shall be uploaded - typically provided by the CI/CD environment."
         type: string
         scope:
           - PARAMETERS
           - STAGES
           - STEPS
         secret: true
         resourceRef:
+          - name: targetRepositoryCredentialsId
```

Comment:
if we expose this via a Jenkins credential, there should also be a proper Vault reference

Reply:
@kingvvgo Can you take over? We just created this PR to show that `helmExecute` does not really work yet on `Jenkins` and that several things are still missing.

Reply:
@c0d1ngm0nk3y  ok, I take over your PR.